dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java

```diff
@@ -160,7 +170,17 @@ public void handle(TaskCallBack taskCallBack) throws TaskException {
 
             // run datax processDataSourceService
             String jsonFilePath = buildDataxJsonFile(paramsMap);
-            String shellCommandFilePath = buildShellCommandFile(jsonFilePath, paramsMap);
+            String shellCommandFilePath;
+            if (dataXParameters.getYarn() == 1) {
```

Comment:
A bit misleading if `yarn` is a numeric type.
Maybe use enum or Boolean to represent whether to use yarn?